www/src/pages/en/usage/first-steps.md

```diff
@@ -0,0 +1,31 @@
+---
+title: First Steps
+description: Getting started with your new T3 App
+layout: ../../../layouts/docs.astro
+---
+
+You just scaffolded a new T3 App and are ready to go. Here is the bare minimum to get your app working.
+
+## Database
+
+If your app includes Prisma, make sure to run `npx prisma db push` from the root directory of your app. This command will sync your Prisma schema with your database and will generate the TypeScript types for the Prisma client based on your schema. Note that you need to restart the TypeScript server after doing this so that it can detect the generated types.
+
+## Authentication
+
+If your app includes Next-Auth, we get you started with the `DiscordProvider`. This is one of the simplest providers that Next-Auth offers, but it still requires a bit of initial setup on your part.
```

Comment:
```suggestion
If your app includes NextAuth, we get you started with the `DiscordProvider`. This is one of the simplest providers that NextAuth offers, but it still requires a bit of initial setup on your part.
```

Reply:
The [NextAuth.js](https://next-auth.js.org/getting-started/example#install-nextauth) documentation uses `NextAuth` instead of `Next-Auth`. We should use the same IMO.
![NextAuth](https://user-images.githubusercontent.com/89210438/200529645-ff5c5833-fb65-4398-afa0-181dc16c425a.png)

Reply:
Looks like the entire docs uses `Next-Auth`, and `NextAuth` interchangeably. I will make a PR later to fix all of them for consistency if that's alright.